core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java

```diff
@@ -118,6 +132,16 @@ protected Builder(OutputStream stream) {
       return this;
     }
 
+    public CsvWriteOptions.Builder ignoreLeadingWhiteSpaces(boolean ignoreLeadingWhitespaces) {
```

Comment:
The s should be lowercase for consistency

Reply:
agreed. Otherwise, it looks great. Thank you